custom_components/tahoma/switch.py

```diff
@@ -112,3 +127,46 @@ async def async_toggle(self, **_):
     def is_on(self):
         """Get whether the switch is in on state."""
         return self.select_state(CORE_ON_OFF_STATE, IO_FORCE_HEATING_STATE) == STATE_ON
+
+
+class TahomaLowSpeedSwitch(TahomaEntity, SwitchEntity, RestoreEntity):
```

Comment:
```suggestion
class TahomaLowSpeedCoverSwitch(TahomaEntity, SwitchEntity, RestoreEntity):
```

Maybe add cover to the name?

Reply:
Done.